ibis/backends/dask/tests/test_dispatcher.py

```diff
@@ -0,0 +1,168 @@
+import pytest
+from multipledispatch import Dispatcher
+from multipledispatch.conflict import AmbiguityWarning
+
+from ..dispatcher import TwoLevelDispatcher
+
+
+class A1(object):
+    pass
+
+
+class A2(A1):
+    pass
+
+
+class A3(A2):
+    pass
+
+
+class B1(object):
+    pass
+
+
+class B2(B1):
+    pass
```

Comment:
i actually don't think this is useful as the dispatcher is already tested in pandas, remove unless there is specific customization (do this generally)

Reply:
I duplicated the tests between pandas and dask so that if the dask pieces ever change we'll know if there's a functionality break. I can drop any tests that are testing pandas functionality if that's your preference though.

Reply:
ok, make a task to factor out the dispatching to a common routine somewhere